includes/cli/class-flush-command.php

```diff
@@ -0,0 +1,78 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-3yuyop.php, but found phpcs-scan-3yuYoP.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).